yt_dlp/extractor/goodgame.py

```diff
@@ -0,0 +1,56 @@
+from .common import InfoExtractor
+from ..utils import clean_html, traverse_obj
+
+
+class GoodGameIE(InfoExtractor):
+    IE_NAME = 'goodgame:stream'
+    _VALID_URL = r'https?://goodgame\.ru/channel/(?P<id>[a-zA-Z0-9_]+)'
+    _TESTS = [{
+        'url': 'https://goodgame.ru/channel/Kochevnik',
+        'info_dict': {
+            'id': 'Kochevnik',
+            'ext': 'mp4',
+            'title': 'Кочетятр',
+            'description': '',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'live_status': 'not_live',
+        },
+        'params': {
+            'skip_download': 'm3u8',
+        },
+        'skip': 'HTTP Error 404: Not Found',
+    }, {
+        'url': 'https://goodgame.ru/channel/LampaRPG',
+        'info_dict': {
+            'id': 'LampaRPG',
+            'ext': 'mp4',
+            'title': r're:Рейтинговые герои.*$',
```

Comment:
With `is_live` set to `True`, timestamp like `2022-05-11 15:54` is appended.

Reply:
```suggestion
            'title': r're:Рейтинговые герои \d{4}-\d{2}-\d{2} \d{2}:\d{2}$',',
```